docs/src/manual/callbacks.md

```diff
@@ -70,6 +70,11 @@ how to use it in the README of their Github repository.
 If you want to modify the problem in a callback, you _must_ use a lazy
 constraint.
 
+ !!! warning
+    You can only set each callback once. Calling `set` twice will over-write the
+    earlier callback. In addition, if you use a solver-independent callback, you
+    cannot set a solver-dependent callback.
```

Comment:
This may be solver specific. Some solvers might support callback that are compatible with solver independent one, e.g. a callback that should return whether to stop

Reply:
This sentence is true for all current solvers implementing callbacks, so I'd prefer to leave as-is for now.